entrypoint.sh

```diff
@@ -5,10 +5,15 @@ mkdir -p /clamav/data
 mkdir -p /clamav/tmp
 cp /etc/clamav/* /clamav/etc/
 
+# Replace values in freshclam.conf
+sed -i 's/^#\?NotifyClamd .*$/NotifyClamd \/clamav\/etc\/clamd.conf/g' /clamav/etc/freshclam.conf
```

Comment:
NotifyClamd has no # at the beginning, so this ensures that both, with and without # replaces the value inside the config.
Imho we should improve this for the other settings too in a separate pull request to make this more reliable against changes to the default from the clamav config settings. Whats your opinion @davosian, does it make sense?

Reply:
@davosian Ready for review again. I will inform you tomorrow around 11:30 CET if it's working, as then a newer version of an antivirus database should be available.

Reply:
Reloading after 120s

```log
2024-10-20 16:19:20 Connected to clamd on tcp://localhost:3310
2024-10-20 16:20:40 RELOADING
2024-10-20 16:20:40 Sun Oct 20 16:20:40 2024 -> Reading databases from /clamav/data
2024-10-20 16:21:18 Sun Oct 20 16:21:18 2024 -> Database correctly reloaded (8699158 signatures)
2024-10-20 16:21:18 Sun Oct 20 16:21:18 2024 -> Activating the newly loaded database...
```

`http://localhost:9000/version`

```json
{
    "Clamav": "1.2.2",
    "Signature": "27433",
    "Signature_date": "Sun Oct 20 10:46:57 2024"
}
```

Reply:
Next antivirus definition

``` console
2024-10-21 11:02:10 Received signal: wake up
2024-10-21 11:02:10 ClamAV update process started at Mon Oct 21 11:02:10 2024
2024-10-21 11:02:10 daily database available for update (local version: 27433, remote version: 27434)
2024-10-21 11:02:14 Testing database: '/clamav/data/tmp.98b9096e0f/clamav-ff2d2d9d28fa91a48f3b7a5a940e7f82.tmp-daily.cld' ...
2024-10-21 11:02:33 Database test passed.
2024-10-21 11:02:33 daily.cld updated (version: 27434, sigs: 2067364, f-level: 90, builder: raynman)
2024-10-21 11:02:33 main.cvd database is up-to-date (version: 62, sigs: 6647427, f-level: 90, builder: sigmgr)
2024-10-21 11:02:33 bytecode.cvd database is up-to-date (version: 335, sigs: 86, f-level: 90, builder: raynman)
2024-10-21 11:02:33 Clamd successfully notified about the update.
2024-10-21 11:02:33 Mon Oct 21 11:02:33 2024 -> Reading databases from /clamav/data
2024-10-21 11:03:17 Mon Oct 21 11:03:17 2024 -> Database correctly reloaded (8699158 signatures)
2024-10-21 11:03:17 Mon Oct 21 11:03:17 2024 -> Activating the newly loaded database...
2024-10-21 11:13:22 Mon Oct 21 11:13:22 2024 -> SelfCheck: Database status OK.
```

http://localhost:9000/version

```json
{
    "Clamav": "1.2.2",
    "Signature": "27434",
    "Signature_date": "Mon Oct 21 10:49:31 2024"
}
```

Reply:
@davosian seems to work correct now